commands/view.js

```diff
@@ -4,7 +4,12 @@ module.exports = async function (context) {
     // grab some features
     const { parameters, strings, print, ignite, filesystem } = context;
     const { pascalCase, isBlank } = strings;
-    const config = ignite.loadIgniteConfig();
+
+    let igniteJson = await filesystem.read(`${process.cwd()}/ignite.json`);
```

Comment:
You could `const { appName, navigation } = await filesystem.read(.... , 'json');` to make it smaller